.github/workflows/botonic-plugin-dashbot-tests.yml

```diff
@@ -21,8 +27,8 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-node-
     - name: Install botonic-plugin-dashbot (Step 4 of 6)
-      run: (cd ./packages/botonic-plugin-dashbot && npm install -D)
+      run: (cd ./packages/$PACKAGE && npm install -D)
     - name: Install common packages dependencies (Step 5 of 6)
       run: npm install -D
     - name: Verify lint botonic-plugin-dashbot (Step 6 of 6)
```

Comment:
please remove "botonic-plugin-dashbot" from the name ( I already removed it from most fpackages, but please check if there are more), so that the workflow files are most similar

Reply:
I created https://github.com/hubtype/botonic/pull/1177/commits/ba06acede628229d3aee2fe5e04eff27f8bcba85 for this